src/cache/prognostic_edmf_precomputed_quantities.jl

```diff
@@ -13,11 +13,13 @@ Updates the edmf environment precomputed quantities stored in `p` for edmfx.
 NVTX.@annotate function set_prognostic_edmf_precomputed_quantities_environment!(
     Y,
     p,
-    ᶠuₕ³,
     t,
 )
     @assert !(p.atmos.moisture_model isa DryModel)
 
+    ᶜuₕ = Y.c.uₕ
+    ᶠu₃ = Y.f.u₃
```

Comment:
Are these used anywhere?

Reply:
I can remove it if it's not, I thought that I passed it to `set_velocity_quantities!`

Reply:
I did not see them used anywhere